pkg/router/functionHandler.go

```diff
@@ -809,7 +809,7 @@ func (fh functionHandler) collectFunctionMetric(start time.Time, rrt *RetryingRo
 	funcMetricLabels.cached = rrt.urlFromCache
 
 	functionCallCompleted(funcMetricLabels, httpMetricLabels,
```

Comment:
Since we are capturing http requests metrics via common metrics package should we get rid of this call ?
`functionCallCompleted`